matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/aggregation/livelocation/LiveLocationAggregationProcessor.kt

```diff
@@ -35,6 +36,7 @@ import timber.log.Timber
 import java.util.concurrent.TimeUnit
 import javax.inject.Inject
 
+// TODO add unit tests
```

Comment:
TODO?

Reply:
Yes it was intented due to lack of time so that it can be merged. It will be added into another dedicated PR.

Reply:
Issue created for that: https://github.com/vector-im/element-android/issues/6155